### ISSUE TITLE
Missing AssemblyVersion attribute in v.0.28.0

### ISSUE THREAD

**Issue by manuelspezzani** [xoofx/markdig#612]
In 0.28.0 the AssemblyVersion attribute seems to be missing or left to its default value 0.0.0.0.

```
[assembly: TargetFramework(".NETFramework,Version=v4.5.2", FrameworkDisplayName = ".NET Framework 4.5.2")]
[assembly: AssemblyCompany("Alexandre Mutel")]
[assembly: AssemblyConfiguration("Release")]
[assembly: AssemblyCopyright("Alexandre Mutel")]
[assembly: AssemblyDescription("A fast, powerful, CommonMark compliant, extensible Markdown processor for .NET with 20+ builtin extensions (pipetables, footnotes, definition lists... etc.)")]
[assembly: AssemblyFileVersion("0.28.0.0")]
[assembly: AssemblyInformationalVersion("0.28.0+983187eace6ba02ee16d1443c387267ad6e78f58")]
[assembly: AssemblyProduct("Markdig")]
[assembly: AssemblyTitle("Markdig")]
[assembly: AssemblyMetadata("RepositoryUrl", "https://github.com/xoofx/markdig")]
[assembly: NeutralResourcesLanguage("en-US")]
[assembly: AssemblyVersion("0.0.0.0")]   <====
```

This could be quite annoying when using binding redirects or when Markdig is a "transitive" dependency, for example on one of my project upgrading to 0.28.0 leads to the dreaded CS1705 build error:

```
Assembly 'Markdig.Wpf' with identity 'Markdig.Wpf, Version=0.5.0.1, Culture=neutral, PublicKeyToken=7b8a8f2a8a0119a0' uses 'Markdig.Signed, Version=0.22.0.0, Culture=neutral, PublicKeyToken=870da25a133885f8' which has a higher version than referenced assembly 'Markdig.Signed' with identity 'Markdig.Signed, Version=0.0.0.0, Culture=neutral, PublicKeyToken=870da25a133885f8'	
```

Thanks a lot.

**Comment by MihaZupan** [comment]
![image](https://user-images.githubusercontent.com/25307628/159280119-d2c203a1-24d0-4491-9a5d-fd3a0d5ff106.png)

![image](https://user-images.githubusercontent.com/25307628/159280062-65c00b29-5432-4ba7-934b-350c91c7c84c.png)

My guess is that this is something `dotnet-releaser` does differently?

**Comment by xoofx** [comment]
Don't think it is `dotnet-releaser` but more likely `MinVer`

**Comment by xoofx** [comment]
This should be fixed by commit ccf455d and available in next version
